packages/story-editor/src/utils/nativeCopyPasteExpected.ts

```diff
@@ -14,28 +14,38 @@
  * limitations under the License.
  */
 
+interface ElementProps extends Element {
+  tagName: string;
+  type?: string;
+  contentEditable: string;
+}
+
 /**
  * Checks if the native copy paste should proceed,
  * either if text is selected or if there's focus on a text/number input.
  *
- * @return {boolean} If native handling is expected.
+ * @return If native handling is expected.
  */
 function nativeCopyPasteExpected() {
   const { activeElement } = document;
-  const { tagName, type, contentEditable } = activeElement;
 
-  // If it's an input in focus, do native handling.
-  if (
-    'true' === contentEditable ||
-    'textarea' === tagName.toLowerCase() ||
-    ('input' === tagName.toLowerCase() &&
-      ['text', 'number', 'search', 'email', 'tel', 'url'].includes(type))
-  ) {
-    return true;
-  }
+  if (activeElement) {
+    const { tagName, type, contentEditable } = activeElement as ElementProps;
```

Comment:
There are default types for elements that have these values, let's use those.
E.g. the activeElement could be a union type, something like
`HTMLInputElement
  | HTMLAnchorElement
  | HTMLTextAreaElement` perhaps?

Reply:
Fixed.